Cargo.toml

```diff
@@ -9,7 +9,6 @@ members = [
   "tokio-fs",
   "tokio-futures",
   "tokio-io",
-  "tokio-macros",
```

Comment:
not sure I understand why this one got removed?

Reply:
Since tokio-macros specifies an edition=2018, which causes failure shown in https://github.com/tokio-rs/tokio/pull/1368#issuecomment-516591998

Reply:
Ok that is fine.